### ISSUE TITLE
Stable releases of the api

### ISSUE THREAD

**Issue by pitsi** [consumet/api.consumet.org#421]
### Describe the bug

The current installation way of the api, which involves cloning the git repo, brings along issues.

### Steps to reproduce

As described in the readme, the "local" way to install the api is to clone the git repo and then use npm. Cloning the repo though gets all all the issues of the development to the user (more in the "actual behavior" part), which is not ideal.

### Expected behavior

After following all the instructions to the letter, the api server should launch and all its providers should work with no issues.

### Actual behavior

However, this is not what happens, at least on my end. Up until the commit of May 4th (= the first time I tried that way of installation), the server launched as usual and all other issues were related to the providers themselves.
Four days and two commits later and the server can not launch at all!

And this made me open this request: please post releases of the api on which the server, i.e.the core component of all this, is in working condition. I know that the providers change constantly and, accorting to the status page, some of them do not work from time to time, but having the server work properly is more important.

### Additional context

The reason behind all this is that I got tired of searching for a working instance, so I decided to host my own eventually. The main one leads to a 1020 error, another one I used in the past leads to a 520 error and a third one no longer exists!
So, as a test, I set up a vm with arch (minimal cli installation + git + npm) and what I mentioned above followed.

**Comment by riimuru** [comment]
@pitsi Apparently the issue started from this #418 pr. The issue has been fixed. I have added a Node.js CI, so from now on there shouldn't be any issues with installations. Thank you for creating this issue!

**Comment by pitsi** [comment]
Sorry for not replying yesterday, but I had a very busy day. Before I start, I want you to know that I am only insterested in flixhq, fmovies and tmdb. I also know that tmdb needs a personal api key to work.

So, thank you for fixing it, but it fails to run npm install today. It just never finishes... And this brings me to the reason I opened this issue.
The main api instance on your server works fine for tmdb, returns a 1020 error for flixhq and... nothing for fmovies, e.g.
```
$ curl -s https://api.consumet.org/movies/fmovies/deadpool
{"statusCode":500,"error":"Internal Server Error","message":"Request failed with status code 403"}
```
Another instance I used in the past, on a different server, returns error 520 for all the providers.

And this led me to host my own instance on my local network, so as to properly test the providers I mentioned above. But with the api server working today and not tomorrow, this is just not doable. That is why I asked for stable releases.
They do not have to be "rock stable, completely bug free, debian stable grade" stable, a "fairly stable, some bugs here and there, arch grade" stable is enough.

p.s. Today I deleted the folder and started over because I wanted to check for a file that is created by npm install.
